motoman_hc10_support/package.xml

```diff
@@ -14,9 +14,10 @@
       <b>Specifications</b>
     </p>
     <ul>
-      <li>HC10 - model 1-06VXHC10-A00</li>
-      <li>HC10DT - model 1-06VXHC10-A10</li>
-      <li>HC10DT-B10 - model 1-06VXHC10-B10</li>
+      <li>HC10 - model 1-06VXHC10-A0*</li>
```

Comment:
Are there other variants than the `A00`? I assume that's why you changed it to `A0*`?